src/scripts/ci/appveyor.yml

```diff
@@ -23,9 +23,12 @@ install:
        call "%ProgramFiles(x86)%\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %PLATFORM%
     )
   - cl # check compiler version
+  
+  - appveyor DownloadFile http://download.qt.io/official_releases/jom/jom.zip -FileName jom.zip
+  - 7z e jom.zip
 
 build_script:
-  - python configure.py --cc=msvc --cpu=%PLATFORM% %MODE%
-  - nmake
+  - python configure.py --cc=msvc --cpu=%PLATFORM% %MODE% --with-pkcs11
+  - jom
```

Comment:
Oh, yeah, idd. You are right. The information is still up to date in the current jom source. This is a strange default given that GNU make uses 1 job if not specified otherwise and jom should be a drop-in replacement for GNU make/nmake.

However, we experienced that in virtual environments like Travis, core detection functions like Linux `nproc` can detect some 32 cores while the power of the virtual machine is limited to 1.5 cores. Do we know what idealThreadCount() returns on AppVeyor? [We get 2 cores there](https://www.appveyor.com/docs/build-configuration#build-environment), so maybe we should fix that number.